pymc_marketing/mmm/evaluation.py

```diff
@@ -0,0 +1,229 @@
+#   Copyright 2024 The PyMC Labs Developers
```

Comment:
the pre-commit is failing because of y=the new year (we need to fix this!)

```
Updated license header in 'tests/mmm/test_evaluation.py'.
Updated license header in 'pymc_marketing/mmm/evaluation.py'.
```

You can simply change 2024 to 2025 as in https://github.com/pymc-labs/pymc-marketing/pull/1326/files 

(sorry for this)